Core/include/Acts/TrackFinding/MeasurementSelector.hpp

```diff
@@ -93,11 +95,25 @@ class MeasurementSelector {
          bool& isOutlier, const Logger& logger) const;
 
  private:
-  template <typename traj_t, typename cut_value_t>
-  static cut_value_t getCut(
-      const typename traj_t::TrackStateProxy& trackState,
-      const Acts::MeasurementSelector::Config::Iterator selector,
-      const std::vector<cut_value_t>& cuts, const Logger& logger);
+  struct InternalCutBin {
+    double maxAbsTheta{};
```

Comment:
```suggestion
    double maxTheta{};
```
&theta; goes from 0 to &pi; [[1](https://acts.readthedocs.io/en/latest/tracking.html#track-parametrization)].